t3/utils/flux.py

```diff
@@ -317,22 +338,32 @@ def run_jsr(gas: ct.Solution,
         V (float, optional): The reactor volume in cm^3.
         a_tol (float, optional): The absolute tolerance for the simulation.
         r_tol (float, optional): The relative tolerance for the simulation.
+        tau_tolerance (float, optional): Cantera will solve JSR equations with step()
+        till tau * tolerance time, after that, it will solve them by advance(t_i).
```

Comment:
Minor: could you space this line and the next one so they start under the word "Cantera" above?